natural-language-understanding/src/main/java/com/ibm/watson/developer_cloud/natural_language_understanding/v1/NaturalLanguageUnderstanding.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.natural_language_understanding.v1;
+
+import com.google.gson.JsonObject;
+import com.ibm.watson.developer_cloud.http.RequestBuilder;
+import com.ibm.watson.developer_cloud.http.ServiceCall;
+import com.ibm.watson.developer_cloud.natural_language_understanding.v1.model.AnalysisResults;
+import com.ibm.watson.developer_cloud.natural_language_understanding.v1.model.ListModelsResults;
+import com.ibm.watson.developer_cloud.natural_language_understanding.v1.model.AnalyzeOptions;
+import com.ibm.watson.developer_cloud.service.WatsonService;
+import com.ibm.watson.developer_cloud.util.GsonSingleton;
+import com.ibm.watson.developer_cloud.util.ResponseConverterUtils;
+import com.ibm.watson.developer_cloud.util.Validator;
+
+/**
+ * Analyze various features of text content at scale. Provide text, raw HTML, or a public URL, and IBM Watson Natural
+ * Language Understanding will give you results for the features you request. The service cleans HTML content before
+ * analysis by default, so the results can ignore most advertisements and other unwanted content.
+ *
+ * ### Concepts Identify general concepts that are referenced or alluded to in your content. Concepts that are detected
+ * typically have an associated link to a DBpedia resource.
+ *
+ * ### Entities Detect important people, places, geopolitical entities and other types of entities in your content.
+ * Entity detection recognizes consecutive coreferences of each entity. For example, analysis of the following text
+ * would count "Barack Obama" and "He" as the same entity:
+ *
+ * "Barack Obama was the 44th President of the United States. He took office in January 2009."
+ *
+ * ### Keywords Determine the most important keywords in your content. Keyword phrases are organized by relevance in the
```

Comment:
I think the text flow here is broken.  The `### Keywords` should be on it's own line since it is the header for the text that follows.  Likewise for the other headers in this block comment.